src/App.res

```diff
@@ -20,12 +19,7 @@ let make = () => {
   let renderFullscreen = switch paymentMode {
   | "paymentMethodCollect" =>
     <LoaderController paymentMode setIntegrateErrorError logger initTimestamp>
-      <React.Suspense
-        fallback={<RenderIf condition={showLoader}>
-          <PaymentElementShimmer />
-        </RenderIf>}>
-        <PaymentMethodCollectElementLazy integrateError logger />
-      </React.Suspense>
+      <PaymentMethodCollectElement integrateError logger />
```

Comment:
Why have we removed the Lazy component?

Reply:
Seemed to be useless at this level (App.res), will add it later for `CollectWidget` component instead